README.md

```diff
@@ -130,6 +131,7 @@
 ### Haskell
 
 * [cabal2nix](https://github.com/NixOS/cabal2nix) - Converts a Cabal file into a Nix build expression.
+* [haskell-flake](https://github.com/srid/haskell-flake) - A [`flake-parts`](https://flake.parts) Nix module for Haskell development.
```

Comment:
Please remove the link from "flake-parts" to make it consistent with the rest. It's otherwise good.

Reply:
Not a wiki. Got it.